bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Title.java

```diff
@@ -58,6 +58,16 @@ default String getType() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the ID for the title component.
+     *
+     * @return the title's ID, if one was set, or predefined.
+     * @since com.adobe.cq.wcm.core.components.models 11.0.0; marked <code>default</code> in 12.1.0
```

Comment:
Version info is off here. Should be 12.4.0. And we don't need to mention the marked default version if it's the same as the since version.